### ISSUE TITLE
When going to a url/href location, how can I place that location on the top of the page with no other content above it?

### ISSUE THREAD

**Issue by ray836** [futurepress/epub.js#1113]
// Goes to exact location instead of file chapter is located in.
      this.adminEpubViewer.epub.goto(this.chosenChapter.href);

![image](https://user-images.githubusercontent.com/24611204/90815769-70bdd300-e2e8-11ea-92a6-e9aa76b9e7ec.png)

Is there a way to make CHAPTER XIV at the top of the page with no content above it (when navigating to it)?

Note: My location looks like: @public@vhost@g@gutenberg@html@files@22012@22012-h@22012-h-2.htm.html#pgepubid00039

**Comment by johnfactotum** [comment]
I don't think this is possible with the `paginated` flow. If you set `flow: 'scrolled'` or `flow: 'scrolled-doc'`, then this should already be the default behavior.
